server/db_setup.sql

```diff
@@ -1,11 +1,15 @@
 DROP Table IF EXISTS laptop_request, laptop_donation, laptop_assignment;
-DROP TYPE IF EXISTS assignment_status;
+DROP TYPE IF EXISTS assignment_status, laptop_request_status;
+
+CREATE TYPE laptop_request_status AS ENUM ('ACTIVE', 'CANCELED');
```

Comment:
To be consistent with existing uses the British spelling `CANCELLED` should probably be used here.

Reply:
Done, sorry for not keeping an eye out for this